var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -84,14 +121,17 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
         when="+raja",
         msg="RAJA implementation requires architecture to be specfied by dir=",
     )
-
+    # depends_on("cuda@12.3.0:", when="+thrust")
     # Thrust Conflict
     # conflicts("~cuda", when="+thrust", msg="Thrust requires +cuda variant")
     depends_on("thrust", when="+thrust")
+    depends_on("cuda", when="+thrust")
+    depends_on("hip", when="+hip")
     depends_on("rocthrust", when="+thrust implementation=rocm")
```

Comment:
I think this dependency needs to be updated now. It looks like the `implementation` variant was removed, so now this dependency is broken.

Reply:
Thank you for spotting this Doug, will push a commit to fix it asap!